lib/mayaUsd/ufe/UsdUndoAttributesCommands.h

```diff
@@ -60,12 +60,13 @@ class UsdAddAttributeCommand : public UsdUndoableCommand<Ufe::AddAttributeComman
         const Ufe::Attribute::Type& type);
 
     Ufe::Attribute::Ptr attribute() const override;
+    void                setName(const std::string& newName);
```

Comment:
I'm hopeful you can remove this method.

Reply:
Or make it private. Feels strange to give programmers the option of changing something that is managed internally by the command.

Reply:
Yes, you are right, not putting it as private is a mistake in this case. I correct it.